### DISCUSSION TITLE
Fixed width on all screens

### DISCUSSION THREAD

**Discussion by steamn** [filamentphp/filament#14662] [Help]
### Package

Panel builder

### Package Version

3.2

### How can we help you?

I need the layout width on all pages globally to be the same as for the desktop version, including on mobile devices with the ability to pinch to zoom. Is there any way to do this without rewriting all the styles? 

As far as I remember, there was some trick with meta viewport, but I can't find a solution.

**Comment by leandrocfe** [comment]
Could you explain more about this?

**Reply by steamn** [reply]
I want to disable responsiveness of admin panel at all, just desktop version for all screens
